src/poetry/publishing/publisher.py

```diff
@@ -100,3 +102,20 @@ def publish(
             client_cert=resolved_client_cert,
             dry_run=dry_run,
         )
+
+    def ask(
+        self,
+        question: Union[str, "Question"],
+        default: Optional[Any] = None,
+        hidden: bool = False,
+    ) -> Any:
+        """
+        Prompt the user for input.
+        """
+        from cleo.ui.question import Question
+
+        if not isinstance(question, Question):
```

Comment:
I'm a little concerned that though this code path isn't used, the fact that passing in a `Question` instance will ignore the `hidden` parameter could be an issue in the future. Why don't we just create the question inline instead of using this helper?